fixtures/epcis/epcisquery.src

```diff
@@ -33,7 +33,7 @@ type Document struct {
 	// The date the message was created. Used for auditing and logging.
 	//
 
-	CreationDate time.Time `xml:"creationDate,attr,omitempty" json:"creationDate,omitempty"`
+	CreationDate soap.XsdDateTime `xml:"creationDate,attr,omitempty" json:"creationDate,omitempty"`
```

Comment:
In Go, the naming convention will be **XSD**DateTime

Reply:
Please find the PR for fixing it here: https://github.com/hooklift/gowsdl/pull/196